### DISCUSSION TITLE
How to change SRID in existing project

### DISCUSSION THREAD

**Discussion by sjib** [QGEP/QGEP#767] [Q&A] [answered]
How can I easily change the SRID (e.g. 21781 (LV03) to 2056 (LV95)) in an existing project if I just want to change the definitions (and already have imported LV95 data in the project (but forgot to change the definitions)?

**Comment by m-kuhn** [comment]
What exactly do you need to change?

* The CRS definition in the QGIS project file
* The CRS definition for the database columns
* Reproject the data within the database

**Comment by urskaufmann** [comment]
In the qgep-schema, e.g. od_cover /situation_geometry, it locks like below. How to Change in every geometry-column to 2056?

-- Column: situation_geometry
-- ALTER TABLE qgep.od_cover DROP COLUMN situation_geometry;

ALTER TABLE qgep.od_cover ADD COLUMN situation_geometry geometry(Point,21781);
COMMENT ON COLUMN qgep.od_cover.situation_geometry IS 'Situation of cover (cover hole), National position coordinates (East, North) / Lage des Deckels (Pickelloch) / Positionnement du couvercle (milieu du couvercle)';

**Comment by sjib** [comment, marked as answer]
- Remove triggers and views depending on the geometry columns
- Change SRID of schema: `ALTER TABLE __my_table__ ALTER COLUMN __geom_column__ TYPE Geometry(_geom_type__,2056) USING Set_SRID(__geom_column__,2056)`
- Dump data only (without value lists!)
- Restore data only in freshly installed empty schema
